addons/website/views/snippets/s_quotes_carousel_minimal.xml

```diff
@@ -0,0 +1,93 @@
+                            <img src="/web/image/website.s_quotes_carousel_demo_image_3" class="s_blockquote_avatar img rounded-circle" alt=""/>
+                            <div class="s_blockquote_author">
+                                <span class="o_small">
+                                    <strong>Jane DOE</strong><br/>
+                                    <span class="text-muted">CEO of MyCompany</span>
+                                </span>
+                            </div>
+                        </div>
+                    </blockquote>
+                </div>
+                <!-- #02 -->
+                <div class="carousel-item o_bg_img_center pt80 pb80" data-name="Slide">
+                    <blockquote data-name="Blockquote" data-snippet="s_blockquote" class="s_blockquote s_blockquote_with_icon o_animable position-relative d-flex flex-column gap-4 w-75 mx-auto my-4 p-5 fst-normal" data-vcss="001">
+                        <div class="s_blockquote_line_elt position-absolute top-0 start-0 bottom-0 bg-o-color-1"/>
+                        <div class="s_blockquote_wrap_icon position-absolute top-0 start-50 translate-middle w-100">
+                            <i class="s_blockquote_icon fa fa-quote-right fa-3x d-block mx-auto text-o-color-2" role="img"/>
+                        </div>
+                        <p class="s_blockquote_quote my-auto h3-fs" style="text-align:center;">" A trusted partner for growth. <br/>Professional, efficient, and always ahead of the curve. "</p>
+                        <div class="s_blockquote_infos d-flex gap-2 flex-column align-items-center w-100 text-center">
+                            <img src="/web/image/website.s_quotes_carousel_demo_image_4" class="s_blockquote_avatar img rounded-circle" alt=""/>
+                            <div class="s_blockquote_author">
+                                <span class="o_small">
+                                    <strong>John DOE</strong><br/>
+                                    <span class="text-muted">CCO of MyCompany</span>
+                                </span>
+                            </div>
+                        </div>
+                    </blockquote>
+                </div>
+                <!-- #03 -->
+                <div class="carousel-item pt80 pb80" data-name="Slide">
+                    <blockquote data-name="Blockquote" data-snippet="s_blockquote" class="s_blockquote s_blockquote_with_icon o_animable position-relative d-flex flex-column gap-4 w-75 mx-auto my-4 p-5 fst-normal" data-vcss="001">
+                        <div class="s_blockquote_line_elt position-absolute top-0 start-0 bottom-0 bg-o-color-1"/>
+                        <div class="s_blockquote_wrap_icon position-absolute top-0 start-50 translate-middle w-100">
+                            <i class="s_blockquote_icon fa fa-quote-right fa-3x d-block mx-auto text-o-color-2" role="img"/>
```

Comment:
I'm wondering if we shouldn't remove `text-o-color-2` so the icon will react to color presets 🤔

It won't adapt to `s_blockquote` color presets anymore, but it's already the case and you seem to remove it in `theme_artists` to achieve that, so maybe it's worth removing it by default. Also, when you click on the snippet, you often land on the carousel container rather than the blockquote one, so I believe chances that you change the carousel presets are higher and we should match that.

What do you think?